sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-ecert-blocked-message.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    16,
+    'Ecert Blocked Student Notification',
```

Comment:
Ecert is a more a federal file description. I would advice the name to be `Disbursement blocked student notification`

Reply:
I agree, with @dheepak-aot . Please rename it for better understanding.

Reply:
@dheepak-aot please keep in mind these messages are supposed to be something to be displayed to the student, similar to _Ministry completes updating exception for an application._.
The student can be something following the email content, for instance: _Disbursement withheld_.
IMO, the suggested ones and the one in the PR seems more like a notification name.